### ISSUE TITLE
Error collecting runner groups for Org

### ISSUE THREAD

**Issue by cjbischoff** [Legit-Labs/legitify#164]
### TL;DR

I received a panic when running on an Org.

LEGITIFY_TOKEN=XXXXXX ./legitify analyze --org ORG

```
panic: interface conversion: interface {} is *github.ListMembersOptions, not *github.ListOptions

goroutine 133 [running]:
github.com/Legit-Labs/legitify/internal/clients/github/pagination.(*ghOptioner).Advance(0xc0005b6000?, {0x2025e00?, 0xc0002a8160?}, {0x1db9ae0?, 0xc00050c040?})
	/home/runner/work/legitify/legitify/internal/clients/github/pagination/pagination.go:19 +0x65
github.com/Legit-Labs/legitify/internal/clients/pagination.(*MappedPager[...]).Async.func1()
	/home/runner/work/legitify/legitify/internal/clients/pagination/mapper.go:51 +0xb1
created by github.com/Legit-Labs/legitify/internal/clients/pagination.(*MappedPager[...]).Async
	/home/runner/work/legitify/legitify/internal/clients/pagination/mapper.go:41 +0xee
```

### Expected behavior

Dunno, first time I used this

### Version

legitify version 0.2.3 commit 66094d78a027db540bd4a49edd62ebc4b2177c25

### On which operating system are you using legitify?

Mac OS

### Relevant log output

```text
~/legitify_0.2.3_darwin_amd64.tar ❯❯❯ cat error.log
2023/03/06 14:21:41 2023/03/06 14:21:41 Error collecting runner groups for ORG - GET https://api.github.com/orgs/ORG/actions/runner-groups: 404 Not Found []
2023/03/06 14:21:41 2023/03/06 14:21:41 Error collecting runner groups for ORG - GET https://api.github.com/orgs/ORG/actions/runner-groups: 404 Not Found []
2023/03/06 14:21:41 2023/03/06 14:21:41 Error collecting runner groups for ORG - GET https://api.github.com/orgs/ORG/actions/runner-groups: 404 Not Found []
2023/03/06 14:21:41 2023/03/06 14:21:41 BUG: closing bar actions although it is not completed. please report this issue to legitify repository.
```

**Comment by gal-legit** [comment]
hey @cjbischoff, thanks for reporting the issue!
We just fixed this bug at #161.
I'm rolling v0.2.4 with the fix, please let me know if the issue reproduces with the new version.

**Comment by cjbischoff** [comment]
@gal-legit Issue sorted with a newer release/Thank you. Ill close
